src/services/Product.service.ts

```diff
@@ -27,3 +27,15 @@ export const findProductService = async (productId: string) => {
   const getProductRequest = await Product.findByPk(productId);
   return getProductRequest;
 };
+
+// Update a product
```

Comment:
```suggestion
```

Reply:
Thank you I am going to do as you suggested.

Reply:
Good morning Axel, 
I have done rebase and I have removed the un necessary changes as you have suggested.
I willing and waiting to listen from you about this changes.